src/views/helpWithHint.njk

```diff
@@ -55,4 +55,29 @@
   type: "submit"
 }) }}
   </form>
+
+  <script>
+    document.addEventListener("DOMContentLoaded", function () {
+      const storedHelpWithHint = localStorage.getItem("helpWithHint");
+      const radioToSelect = document.querySelector('input[value=' + storedHelpWithHint + ']');
+
+      if (storedHelpWithHint && radioToSelect) {
```

Comment:
I think you can simplify the code here by removing the "if" and set the return of this condition directly to the variable you want to set.
Why? because "if (true)" help to set to true, then yourvariable = if (true);
That will give you this: radioToSelect.checked = (storedHelpWithHint && radioToSelect);

Reply:
good suggestion! i'll amend